### PR TITLE
Branch add view command

### LINKED ISSUE #11: As a financial advisor, I can view a specific client profile
...so that I can find and view a specific client.

### PR DESCRIPTION
**Pull request by sethteo**
Fixes #11

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/ViewCommand.java
Comment (itstrueitstrueitsrealitsreal) [inline src/main/java/seedu/address/logic/commands/ViewCommand.java:1]: Good use of SLAP in this file!

File: src/test/java/seedu/address/logic/commands/AddCommandTest.java
Comment (itstrueitstrueitsrealitsreal) [inline src/test/java/seedu/address/logic/commands/AddCommandTest.java:1]: Good to see that you've updated tests as well.

File: src/test/java/seedu/address/logic/parser/ViewCommandParserTest.java [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+import static seedu.address.logic.parser.CommandParserTestUtil.assertParseSuccess;
+import static seedu.address.testutil.TypicalIndexes.INDEX_FIRST_PERSON;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.logic.commands.ViewCommand;
+
+/**
+ * As we are only doing white-box testing, our test cases do not cover path variations
+ * outside of the DeleteCommand code. For example, inputs "1" and "1 abc" take the
```
Comment (itstrueitstrueitsrealitsreal) [inline src/test/java/seedu/address/logic/parser/ViewCommandParserTest.java]: Remember to change DeleteCommand to ViewCommand!

File: src/test/java/seedu/address/testutil/PersonBuilder.java [hunk 2]
```diff
@@ -1,8 +1,13 @@
 package seedu.address.testutil;
 
+import static seedu.address.testutil.TypicalPersons.CHAD;
+import static seedu.address.testutil.TypicalPersons.JAMAL;
```
Comment (itstrueitstrueitsrealitsreal) [inline src/test/java/seedu/address/testutil/PersonBuilder.java:4]: Your test names are so funny!

**Review by itstrueitstrueitsrealitsreal — APPROVED** [review]
Good effort adding in the view command and adding test cases to remove the occurence of bugs!

File: src/test/java/seedu/address/logic/parser/ViewCommandParserTest.java (hunk 1 above)
Comment (sethteo) [reply src/test/java/seedu/address/logic/parser/ViewCommandParserTest.java]: Thanks for spotting this!

**Pull request merged**